### PR TITLE
feat(chip): added squared prop

### PR DESCRIPTION
**Pull request by Emacdyz**
# Description

With the genre filter coming up, design requested a squared version of the ChipWithToggle

## How to test

- Checkout this branch
- `$ yarn storybook`
- go to actions > chip > With toggle squared corners and check it looks different (more squared) but still work as the more rounded one 

## Links
https://ticketswap.atlassian.net/browse/APO-2611

### CODE REVIEW

File: src/components/Chip/Chip.tsx [hunk 1]
```diff
@@ -26,13 +26,14 @@ interface ContainerProps {
 const Container = styled.span<ContainerProps>`
   --_accentColor: ${color.inactive};
   --_backgroundColor: ${color.background};
+  --_borderRadius: ${space[32]};
```
Comment (RobVermeer) [inline src/components/Chip/Chip.tsx]:
```suggestion
  --_borderRadius: ${({ squared }) => squared ? space[8] : space[32]};
```

I think this is a bit more clean, then you don't need a custom styled component. You do have to pass the squared prop to the main chip component, but I think this also just makes sense, so all the chip's can be "squared".

**Comment by RobVermeer** [comment]
Do you have an example of a design with this squared chip? 🤔 To me it sounds a bit weird to have a chip which looks more like a normal button (why do we have a chip then??) 🤔 

Btw if this is a one-of, then the change to make it a customizable css variable, would be enough as well. So then there is no need to add a prop, just style the component inside Sirius (might not what we want right now though).

**Comment by Emacdyz** [comment]
@RobVermeer, so Mark just told me that they made some updates and want all the chips to be more rectangelish, so that simplifies this PR!

**Review by viktorrenkema — APPROVED** [review]
🔴

**Review by bobbonius — APPROVED** [review]

**Pull request merged**